metrics/node_id.go

```diff
@@ -4,6 +4,19 @@ import (
 	"strconv"
 )
 
+const (
+	trueStr  = "true"
+	falseStr = "false"
+)
+
 func idToString(id uint32) string {
 	return strconv.FormatUint(uint64(id), 10)
 }
+
+func nodeHintHitString(preferred, actual string) string {
```

Comment:
you can also simply use strconv.FormatBool()

Reply:
done that